www/include/configuration/configObject/servicegroup/DB-Func.php

```diff
@@ -295,14 +469,27 @@ function updateServiceGroupServices($sg_id, $ret = array(), $increment = false)
         for ($i = 0; $i < count($retTmp); $i++) {
```

Comment:
could you move the prepared statement of lines 479 and 486 before this loop. It will require lesser resources on DB